package.json

```diff
@@ -28,7 +28,9 @@
     "url": "https://github.com/HubSpot/general-store/issues"
   },
   "homepage": "https://github.com/HubSpot/general-store",
-  "dependencies": {},
+  "dependencies": {
+    "immutable-is": "^3.7.4"
```

Comment:
If `immutable-is` is baked into the builds, then this is actually a devDependency not a dependency.